lib/Dialect/TTIR/Transforms/Passes.cpp

```diff
@@ -516,8 +525,9 @@ class TTIRLayoutFuncReturnRewriter
     for (auto &operand : op->getOpOperands()) {
       // Leave the return values in initMemorySpace, optimizer might decide
       // otherwise
+      bool tiled = false;
```

Comment:
I just made it a named variable instead of doing `/*tiled*/false` as the function argument. This is the desired format for outputs, I.e. programming what the to_layout should do.

I guess I'm not following how the assert should read, it doesn't matter what layout the return operands are in before this to_layout construction.

Reply:
Ack